keel-test/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/BaseClusterHandlerTests.kt

```diff
@@ -0,0 +1,96 @@
+package com.netflix.spinnaker.keel.api.plugins
+
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.api.ResourceDiff
+import com.netflix.spinnaker.keel.api.ResourceSpec
+import com.netflix.spinnaker.keel.api.support.EventPublisher
+import com.netflix.spinnaker.keel.time.MutableClock
+import dev.minutest.junit.JUnit5Minutests
+import dev.minutest.rootContext
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import strikt.api.expectThat
+import strikt.assertions.isFalse
+import strikt.assertions.isTrue
+import java.time.Clock
+
+abstract class BaseClusterHandlerTests<
```

Comment:
It feels to me like this class is in the wrong module. The `keel-test` module was intended to be just common testing things, and `keel-core-test` would contain base classes for common tests (e.g. the repository tests).

Reply:
i'll move it.